core/src/win32/plugins/filed/mssqlvdi-fd.cc

```diff
@@ -1628,6 +1628,10 @@ static bRC end_restore_job(PluginContext* ctx, void* value)
  */
 static bRC startRestoreFile(PluginContext* ctx, const char* cmd)
 {
+  // If no explicit server address given use the DEFAULT_SERVER_ADDRESS.
+  if (!p_ctx->server_address) {
```

Comment:
`p_ctx` is not declared here.
We could simple ditch that commit and merge the documentation fixes if you want to.

Reply:
I agreed. We have your PR for the Code Changes.